giddy/markov.py

```diff
@@ -214,6 +250,12 @@ class Spatial_Markov(object):
                       discretization.
     variable_name   : string
                       name of variable.
+    fill_diag       : bool
```

Comment:
Thinking about this, I think `fill_diag` might be a bit confusing of terminology, especially compared to what that means elsewhere in `pysal` & `numpy`. This only fills diagonals where the row sums are zero. Normal `fill_diagonal` stuff fills all elements. 

What do you think about `fill_empty_classes` or `fill_empty`?

Reply:
`fill_empty_classes` sounds much better! @sjsrey what do you think?

Reply:
Commit https://github.com/pysal/giddy/pull/109/commits/0676528b415553dc043ea33b02bc89600d5edba9 has renamed `fill_diag` to `fill_empty_classes`